docs/users.rst

```diff
@@ -302,8 +302,6 @@ An example::
     - nss_wrapper  # package name, not an NVR.
     - httpd
     - httpd-devel
-    # used for requesting ODCS compose of type "pulp"
-    pulp_repos: true
```

Comment:
Do we want to remove the pulp references from this file (`docs/users.rst`)? These options are the ones passed to ODCS for the composes.

Reply:
We need to keep the `pulp_repos` option. Please revert this part.